.devcontainer/Dockerfile

```
@@ -1,9 +1,9 @@
-RUN npm install -g yarn && \
-    sudo apt-get update && sudo apt-get install gcc g++ && \
-    mkdir -p /root/.hydro /data/file && chmod +x /usr/bin/sandbox && \
-    chown -R root:root /root /root/.hydro /data/file && \
-    echo '{"uri":"mongodb://mongo/hydro"}' > /root/.hydro/config.json && \
-    echo '["@hydrooj/ui-default","@hydrooj/hydrojudge"]' > /root/.hydro/addon.json
-ENTRYPOINT sandbox
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:22-bookworm
+
+RUN install -d -o root -g root /root/.hydro /data/file \
+    && cat <<'EOF' > /root/.hydro/config.json
+{"uri":"mongodb://mongo/hydro"}
```

Hadolint (2.12.0)

[error] unexpected '{'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)